packages/app/src/messages/components/messages/messages.component.tsx

```diff
@@ -147,6 +149,16 @@ const Messages: FunctionComponent<MessagesProps> = ({
   const [activeSearchDropdown, setActiveSearchDropdown] =
     useState<boolean>(false)
   const allItemsSelected = threads.length === selectedItems.rows.length
+  const showThreadDeletedPopup = threadPopupNotifications.length > 0
+  const showMessageDeletedModal = messagePopupNotifications.length > 0
+
+  const removePopupNotification = (notifications: Notification[]) => {
+    return notifications
```

Comment:
small thing, but no need to return.

Reply:
removed in [cdb4aea](https://github.com/mudita/mudita-center/pull/996/commits/cdb4aead3df31d3ade0b8b629cc868077614298c)